packages/transport-grpc/src/unary.js

```diff
@@ -4,24 +4,41 @@ import {NodeHttpTransport} from "@improbable-eng/grpc-web-node-http-transport"
 
 grpc.setDefaultTransport(NodeHttpTransport())
 
-export async function unary(host, method, request, context) {
+export async function unary(
+  host,
+  method,
+  request,
+  context,
+  retryLimit = 5,
+  retryIntervalMs = 1000
+) {
   invariant(
     context.config,
     `SDK GRPC Unary Error: context.config must be defined.`
```

Comment:
Especially confusing if the error is spitting that out...but that's coming from grpc?

Reply:
from `send` I thnk, no? `{config, response, ix: ixModule, Buffer},`